web-wallet/src/lib/stores/__tests__/walletStore.spec.js

```diff
@@ -26,6 +26,7 @@ import { walletStore } from "..";
 describe("Wallet store", async () => {
   vi.useFakeTimers();
 
+  const AUTO_SYNC_INTERVAL = 5 * 60 * 1000;
```

Comment:
Should we extract this to a const file that's shared in the app? We have it in the test and the walletStore itself.